tests/compiler/simd.ts

```diff
@@ -188,6 +188,91 @@ function test_v128(): void {
     assert(v == i32x4(42, 0, 0, 0));
     __free(ptr);
   }
+  {
+    let ptr = __alloc(16);
+    store<i32>(ptr, 42);
+    let v: v128 = v128(0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0);
+    v = v128.load_lane<i32>(ptr, v, 0);
+    assert(v == i32x4(42, 0, 0, 0));
+    v = v128.load8_lane(ptr, v, 0);
+    assert(v == v128(42, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0));
+    v = v128.load16_lane(ptr, v, 0);
+    assert(v == i16x8(42, 0, 0, 0, 0, 0, 0, 0));
+    v = v128.load32_lane(ptr, v, 0);
+    assert(v == i32x4(42, 0, 0, 0));
+    v = v128.load64_lane(ptr, v, 0);
+    assert(v == i64x2(42, 0));
+    __free(ptr);
+  }
+  {
+    let v: v128 = v128.ceil<f32>(f32x4(1.1, -0.25, 70.01, 4.0));
+    assert(v128.extract_lane<f32>(v, 0) == 2.0);
+    assert(v128.extract_lane<f32>(v, 1) == 0.0);
+    assert(v128.extract_lane<f32>(v, 2) == 71.0);
+    assert(v128.extract_lane<f32>(v, 3) == 4.0);
+  }
+  {
+    let v: v128 =v128.floor<f32>(f32x4(1.1, -0.25, 70.01, 4.0));
+    assert(v128.extract_lane<f32>(v, 0) == 1.0);
+    assert(v128.extract_lane<f32>(v, 1) == -1);
+    assert(v128.extract_lane<f32>(v, 2) == 70.0);
+    assert(v128.extract_lane<f32>(v, 3) == 4.0);
+  }
```

Comment:
please use `== f32x4(1, -1, 70, 4)` instead extract_lane

Reply:
@MaxGraey 
for the function `floor`, there is no issue, but for `ceil`, it is not equal to `f32x4(2, 0, 71, 4)`.
here is my test code:  
```
export function test():bool {
    let v: v128 = f32x4.ceil(f32x4(1.1, -0.25, 70.01, 4.0));
    return (v == f32x4(2, 0, 71, 4));
}
```

Reply:
Perhaps it should be `f32x4(2, -0.0, 71, 4)` ?

Reply:
Yes, it is -0.0